portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +30,15 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function toggleVisibility (sectionToToggle) {
+  const invisibleClass = 'invisible';
+  const toggleDiv = document.getElementById(sectionToToggle);
+  const otherDiv = document.getElementById(sectionToToggle === 'swimming' ? 'rowing': 'swimming');
+  if (toggleDiv.classList.contains(invisibleClass)) { //toggleDiv becomes visible, all others become invisible
```

Comment:
super nit: generally it's better for these inline comments to go above or below an if statement line, but not the same